src/main/java/uk/gov/hmcts/darts/cases/helper/AdvancedSearchRequestHelper.java

```diff
@@ -150,12 +150,17 @@ private List<Predicate> addCourthouseCriteria(GetCasesSearchRequest request, Cri
 
     private List<Predicate> addUserSecurityRolesCriteria(CriteriaBuilder criteriaBuilder, Root<CourtCaseEntity> caseRoot) {
         List<Predicate> predicateList = new ArrayList<>();
+        Join<CourthouseEntity, SecurityGroupEntity> securityJoin = joinSecurityGroup(caseRoot);
         Join<CourtCaseEntity, UserAccountEntity> userJoin = joinUser(caseRoot);
-        predicateList.add(criteriaBuilder.equal(
-                              criteriaBuilder.lower(userJoin.get(UserAccountEntity_.EMAIL_ADDRESS)),
-                              userIdentity.getUserAccount().getEmailAddress().toLowerCase()
-                          )
+
+        Predicate predicateGlobalAccess = criteriaBuilder.equal(
```

Comment:
it looks like this would just check to see if any security groups exist that have global access, but doesn't check to see if they are related to this user.